src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+/* code here */
+@import "../utils/mixins";
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  &:nth-child(-n + 4) {
+    @include margin-nth-child;
+
+    margin-right: $card-margin-right;
+  }
+
+  &:nth-last-child(-n + 4) {
+    @include margin-last-child;
+
+    margin-right: $card-margin-right;
+  }
+
+  &:nth-child(4n) {
+    @include margin-r-last-child;
+  }
+
+  &__photo {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__description {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+  &__code {
+    margin-bottom: 16px;
+    align-self: flex-start;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+  &__rating {
+    width: 166px;
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__rating-reviews {
+    justify-self: center;
+    align-self: center;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-main-accent;
+  }
+  &__cost {
+    width: 100%;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__cost-text {
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+    order: 0;
+  }
+  &__cost-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    width: 166px;
+    height: 40px;
+    background: $color-accent-blue;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      border-color: $color-accent-blue;
+      background-color: #fff;
```

Comment:
Use var

Reply:
for what? if you mean - color, i not agree. because this color i used only once

Reply:
Not once

Reply:
![image](https://user-images.githubusercontent.com/110053436/218108433-6bedb617-6534-4339-90b6-8d154b77a846.png)
ok, two times. but really we need to use var in same situations?